staging/src/k8s.io/kubectl/pkg/cmd/apply/apply_test.go

```diff
@@ -2206,10 +2208,102 @@ func TestApplySetParentValidation(t *testing.T) {
 	}
 }
 
+func setUpClientsForApplySetWithSSA(t *testing.T, tf *cmdtesting.TestFactory, objects ...runtime.Object) {
```

Comment:
This is extracted from `TestApplySetParentManagement` after adjusting it for the use of the unstructured client. Four different ApplySet tests need all (or most of) this same setup. The only ApplySet tests that don't need it are the ones that don't run the whole command (e.g. test early errors), or are the CSA-based tests Justin wrote (which need more / different 'endpoints' than this provides).